src/Common/src/Interop/Unix/System.Security.Cryptography.Native/Interop.EcKey.cs

```diff
@@ -38,7 +38,7 @@ internal static int EcKeyGetSize(SafeEcKeyHandle key)
             throw Interop.Crypto.CreateOpenSslCryptographicException();
         }
 
-        [DllImport(Libraries.CryptoNative)]
+        [DllImport(Libraries.CryptoNative, EntryPoint = "CryptoNative_EcKeyGetCurveName")]
```

Comment:
Yeah I was flipping between a 2 and no 2 in order to try to emulate the centos errors, and accidently committed without the 2